config/notification_banner.yml

```diff
@@ -7,18 +7,18 @@ localhost:
 
 development:
   notification:
-    message: This service will be unavailable on Wednesday 28 August 2024 between 6pm and 8pm.
-    date_from: "2024-08-27T18:00:00"
-    date_to: "2024-08-28T19:00:00"
+    message: This service will be unavailable on Wednesday 4 September 2024 between 6pm and 8pm.
+    date_from: "2024-08-30T16:00:00"
+    date_to: "2024-09-04T19:00:00"
```

Comment:
should that be "2024-09-04T20:00:00" if its 8pm?

Reply:
its in UTC, but agreed it could be clearer, other than the comment at the top pf the file